sdk/monitor/monitor-opentelemetry/src/metrics/quickpulse/utils.ts

```diff
@@ -63,7 +67,7 @@ export function getSdkVersion(): string {
   return internalSdkVersion;
 }
 
-/** Set the version prefix to a string in the format {ResourceProvider}{OS}m_ */
+/** Set the version prefix to a string in the format "\{ResourceProvider\}\{OS\}m_ */
```

Comment:
Doesn't really make sense to escape these chars in a comment explaining formatting.

Reply:
Yes, I suppressed the linting errors instead